### DISCUSSION TITLE
This client isn't compatible with the media and the server isn't sending a compatible media format.

### DISCUSSION THREAD

**Discussion by JobberRT** [jellyfin/jellyfin#11098] [Q&A] [answered]
## I think this is a bakcend problem so I create one here as well.
The source post was on `jellyfin-web`: https://github.com/jellyfin/jellyfin-web/discussions/5249

**Comment by TimGels** [comment]
To start, we would need logs from FFMPEG and normal server logs.

**Reply by JobberRT** [reply]
Hi, the detailed info, including ffmpeg log, was attached to the `jellyfin-web` repo, I will regenerate a new one after work and post them here, thanks

**Reply by TimGels** [reply]
My bad. I overlooked the log there. The best place is indeed in the form of an issue.

**Comment by JobberRT** [comment]
## System Information
CPU: 12400
RAM: 4GB
DISK: 64GB
Running in PVE VM with SR-IOV vGPU passthrough. Validated vGPU is working by playing other media file

## FFmpeg And Jellyfin Log
[jellyfin.txt](https://github.com/jellyfin/jellyfin/files/14496364/jellyfin.txt)
[ffmpeg.txt](https://github.com/jellyfin/jellyfin/files/14496365/ffmpeg.txt)


## Jellyfin HWA Setting Screenshot
**Only the part that I changed and sorry for not changing it to English, I tried in the webui and force clear the cache, but it didn't help**


![1](https://github.com/jellyfin/jellyfin-web/assets/17498767/f84f24e6-5cce-46dd-b06b-0598b4771470)
![2](https://github.com/jellyfin/jellyfin-web/assets/17498767/6b879c10-9e02-41e5-aee2-366698d325f6)


### I don't know what problem here I have, but I can play some other movies normally. I'm software engineer so I can provide detailed information and I want to, please tell me if you need anything else

**Comment by JobberRT** [comment]
@TimGels Sorry for the late reply, Too busy with my work yesterday.

The system spec are copied to here to make it eaasy to read. And the jellyfin/ffmpeg log also attached. From `ffmpeg` log I can't see anything useful,  but from `jellyfin` log I see `Current HLS implementation doesn't support non-keyframe breaks but one is requested, ignoring that request` error

**Comment by JobberRT** [comment]
It seems to be a transcode error, but even I disable HWA this issue is still there

**Comment by JobberRT** [comment, marked as answer]
For anyone who has the same problem, take a look at [#5039](https://github.com/jellyfin/jellyfin-web/issues/5039) . I'm reducing my audio channel number as a workaround and it works.
